### PR TITLE
Use all default image block features in its template

### PR DESCRIPTION
**Pull request by bethfraser**
## Sources
We've had a couple of bug reports lately where the hyperlink option for image blocks hasn't been included, so users add a link and then it doesn't get added to the page - e.g. https://app.forecast.it/project/P-110/workflow/T5497. This is probably because it's not in the template! 

## Description
We include the options to add a link, description/alt text and 'open in new window' in the block by default so I have added these to the template. 

I've also updated the image's default Wagtail resizing property, which used to be fill-600x600. I think it's very unlikely that we'll have any project where we want the default image block to always be cropped to a square, so I've changed it to max-1024x768 which I think is a more reasonable default to start with. 

## Setup instructions
- Install cookiecutter if you don't have it with `pip install --user cookiecutter`
- pull this branch of the project, but then navigate to one directory above it (probably your Sites directory)
- create a new project based off this branch by doing `cookiecutter django-template`
- check that this new version of the template has pulled through
- navigate into your test project, run `make reset` and then `manage.py runserver`

## How to test
- Go to 127.0.0.1:8000/admin/ and add an image to the homepage, and check that adding a link URL and alt text works, as well as the 'open in new window' option.

For comprehensive guidelines on code review at DEV, see here: https://www.notion.so/developersociety/Code-Review-b6d646b1d6e54011ada8169aee543231.

### CODE REVIEW

File: {{cookiecutter.project_slug}}/templates/blocks/image_block.html [hunk 1]
```diff
@@ -1,7 +1,13 @@
 {% load wagtailimages_tags %}
 
 <figure>
-  {% image self.image fill-600x600 %}
+  {% if self.link %}
+    <a href="{{ self.link }}" {% if self.new_window %}target="_blank"{% endif %}>
+      {% image self.image max-1024x768 alt=self.description %}
```
Comment (tomkins) [inline {{cookiecutter.project_slug}}/templates/blocks/image_block.html:6]: One of the repeating things we get from partners is: I've set a focal point, and it does nothing!

Looking at: https://docs.wagtail.io/en/v2.11.3/topics/images.html

Focal point is only used on `fill-` images. By switching to max as default, my instinct is that we'll face this question more often.

**Review by tomkins — CHANGES_REQUESTED** [review]
Thanks for getting this back to the template.

I've got a question, however feel free to say if max might be better.

File: {{cookiecutter.project_slug}}/templates/blocks/image_block.html (hunk 1 above)
Comment (bethfraser) [reply {{cookiecutter.project_slug}}/templates/blocks/image_block.html:6]: Hmm, very good point. I'll have a think about this on Monday!

Comment (bethfraser) [reply {{cookiecutter.project_slug}}/templates/blocks/image_block.html:6]: I think actually for image blocks this should be fine - the images a partner should be putting into an image block should already be cropped to show the area they're after before being uploaded. The focal point/cropping should only be being used for things like header images, images on cards or in list views. Does that make sense?

Comment (tomkins) [reply {{cookiecutter.project_slug}}/templates/blocks/image_block.html:6]: 👍 - let's make sure we remember that knowledge for those sorts of cases in PRs!

**Review by tomkins — APPROVED** [review]

**Pull request merged**